CHANGELOG.md

```diff
@@ -1,215 +1,281 @@
 # Changelog
+
 All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
-- No changes yet.
+
+* No changes yet.
+
+## [1.12.0] - 2018-04-23
+
+### Added
+
+* Gen: Added a public interface `ThriftPackageImporter` that determines
+  the import paths from a thrift root.
+
+### Changed
+
+* Gen: Now the constructor `NewGenerator` use a public interface instead of the
+  private implementation allowing it to be used from others packages.
 
 ## [1.11.0] - 2018-03-27
+
 ### Added
-- Plugins now have access to service and function annotations.
+
+* Plugins now have access to service and function annotations.
```

Comment:
Revert these changes please. We shouldn't be rewriting the changelog as part of this PR.

Also, for your change, all you need to do is add these entries under `Unreleased`. The version number, release date, etc. will be set when this change is actually released.

Reply:
done